synapse/tests/test_model_geospace.py

```diff
@@ -428,3 +431,14 @@ async def test_geojson(self):
 
             opts = {'vars': {'geojson': geojson2}}
             nodes = await core.nodes('[ geo:place=* :geojson=$geojson ]', opts=opts)
+
+    async def test_geo_dist_offset(self):
```

Comment:
just roll this into an existing test which loads GeoTstModule

Reply:
No other test method has a name that could logically include this. Unless you're saying we should rename another test to be more generic, i dont see how this is reasonable.